vagrant/bootstrap.sh

```diff
@@ -126,7 +126,7 @@ echo "dbms.shell.enabled=true" >> /etc/neo4j/neo4j.conf
 
 if [ "$VAGRANT" = true ]; then
   # Expose Neo4j to the host, for development purposes
-  echo "dbms.connectors.default_listen_address=0.0.0.0" >> /etc/neo4j/neo4j.conf
+  sudo echo "dbms.connectors.default_listen_address=0.0.0.0" >> /etc/neo4j/neo4j.conf
```

Comment:
Why sudo here? Vagrant runs this script as root already.

Reply:
The script should run as the SSH user, which is now vagrant after some of the early January changes to the ubuntu cloud images.  This sudo makes everything consistent with other commands that might require it now.

Reply:
No, the whole script runs with sudo, so this is redundant. See:
https://www.vagrantup.com/docs/provisioning/shell.html#privileged

Reply:
Aware, but this is the behavior that led me to file #531.  I don't seem to be able to recreate with newer versions of the cloud image, but the symlinks into systemd were inheriting permissions as vagrant:vagrant, leading to failures to init and run the services.  Prefacing those (and this) command with sudo corrected the issue.  I'm not sure why this PR has that remanent, though, so I've pulled it.

I was not able to repro this morning on 20180215.0.0, so perhaps it was transient, but I'm not making it up, I swear. :)